README.md

```diff
@@ -84,16 +84,6 @@ __Objective-C__
 }
 ```
 
-__iOS 14 IDFA access__
-
-In iOS 14 Apple introduces changes to the way the Advertising Identifier (IDFA) can be accessed. In particular, developers are forced to ask the user permission to access the IDFA:
-- https://developer.apple.com/app-store/user-privacy-and-data-use/
-- https://developer.apple.com/documentation/apptrackingtransparency?language=objc
-
-The [Examples](https://github.com/velocidi/velocidi-ios-objc-sdk/tree/master/Examples) folder contains two example applications ready for iOS 14, which ask for user permission before using the IDFA to execute any requests.
-
-The Velocidi iOS SDK itself will **not** execute any request if the user did not allow tracking to proceed.
-
```

Comment:
We should give more detail on why we don't resort to any kind of internal ID and why it is up to the developer to provide us with the ID it wants. 

We should also document how to fetch the IDFA and/or IDFV, given that those would be the most important ones (I think this will come in a later stage eventually, so we can do this in a different PR)

Reply:
In 175d1db I correct the documentation about match and ask for caution when using the IDFA.